terraform/projects/app-ecs-services/nginx-auth-proxy.tf

```diff
@@ -49,7 +49,7 @@ data "template_file" "nginx-auth-proxy-config-file" {
     alertmanager_3_dns_name = "${data.terraform_remote_state.app_ecs_albs.alerts_private_record_fqdns.2}"
     prometheus_1_dns_name   = "${data.terraform_remote_state.app_ecs_albs.prom_private_record_fqdns.0}"
     prometheus_2_dns_name   = "${data.terraform_remote_state.app_ecs_albs.prom_private_record_fqdns.1}"
-    prometheus_3_dns_name   = "${data.terraform_remote_state.app_ecs_albs.prom_private_record_fqdns.2}"
+    prometheus_3_dns_name   = "prom-ec2-3.${data.terraform_remote_state.infra_networking.private_subdomain}:9090"
```

Comment:
it's a pedantic point, but this thing isn't a dns name because it has a port number.  I'm not sure what it should be. `address`, maybe? That fits prometheus's use of the `__address__` label.

Reply:
I encourage pedantic comments 👍 I agree. Will look to get this tidied up when we move the other two prometheis over to EC2